results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+Participants who could not easily see or could not see the staus indicators at all (since they are small and relatively low contrast) relied on prior experience with notebooks, gut instinct about how long that section of code usually takes to run, or the appearance of a cell output to check a cell’s status. Having at least one non-visual option (like an audio indicator) or a visual option that would represent itself in a timely manner for assistive tech (like a notification) could provide users with the support they want. Making these options customizable would be important as well.
+
+#### Flexible command palette search/filter behavior
+
+Because several participants relied heavily on the command palette to complete tasks, the limit of its filtering arose several times. From the user end, the command palette feels like it searches only for direct matches within the command list for whatever the user types in. If a user knows exactly what they are looking for, they often have success. Participants were sometimes able to quickly and flawlessly locate a desired command. But when they were exploring JupyterLab’s command options, unable to remember the exact name of a command, made a typo, or were unfamiliar with a task many participants had to give up using the command palette in order to complete the task. Depending on the assistive tech they use, typos can also be nontrivial to detect for blind or visually impaired users. Having a fuzzy search or other typo/error support rather than only the direct filter would help the command palette continue to be the boon many participants wanted it to be.
+
+## For future tests
+
+Reflecting on the findings and gaps in this set of tests, the following are questions to revisit in the future.
+
+- How might a screen reader user complete common notebook tasks in JupyterLab?
+- What are JupyterLab’s strengths and weaknesses when it comes to authoring notebooks and other documents with assistive tech?
+- What more can we learn about navigating JupyterLab, especially with a variety of assistive tech? What are good examples of navigation that support disabled users?
+- What are the full range of notebook cell output types and what feedback to disabled users have on each type?
+- What is the real time collaboration experience in JupyterLab like for assistive tech users? This question requires JupyterLab collaboration modes to be more stable before it can be tested.
```

Comment:
This seems to come out of nowhere.

Reply:
That makes sense; it isn't mentioned in these tests or related work. I put it because collaboration modes have been a discussion in the community for a long time now and seems to hold a lot of interest for people. I wanted to point out that we have not done a lot of user experience work for the feature(s) that have been added for collaboration, let alone accessibility-focused user experience. 

If it seems too jarring, it can be removed.

Reply:
```suggestion
```